src/components/SelectionList/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { Text } from 'rebass'
```

Comment:
Move content to `src/components/SelectionList/SelectionList.tsx` and export everything:

```
// file src/components/SelectionList/index.ts
export * from './SelectionList`